src/core/components/checkbox/stories/checkbox/required.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { ThemeProvider } from "emotion-theming"
+
+import { CheckboxGroup, Checkbox, checkboxDefault } from "../../index"
+
+const required = () => (
+	<form onSubmit={e => e.preventDefault()}>
+		<ThemeProvider theme={checkboxDefault}>
+			<CheckboxGroup name="emails">
+				<Checkbox
+					label="Accept terms"
+					value="accept_terms"
+					required={true}
+				/>
+			</CheckboxGroup>
+		</ThemeProvider>
+		<button type="submit">Submit</button>
+	</form>
+)
+
+required.story = {
+	name: "required",
+}
+
+const optional = () => (
+	<form onSubmit={e => e.preventDefault()}>
+		<ThemeProvider theme={checkboxDefault}>
+			<CheckboxGroup name="emails">
+				<Checkbox
+					label="Accept terms"
+					value="accept_terms"
+					required={false}
+				/>
+			</CheckboxGroup>
+		</ThemeProvider>
+		<button type="submit">Submit</button>
+	</form>
+)
+
+optional.story = {
+	name: "optional",
+}
```

Comment:
Since optional is the default, we probably don't need another story demonstrating it. Can we remove it?